simulationconfig.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2022-, Semiotic AI, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+
+from simulation import simulation_ingredient
+
+
+@simulation_ingredient.config
+def config():
+    nproducts = 1
```

Comment:
what are products in here? like types of queries?

do we need to have it specified apriori - cannot it come from the customers directly? (asking as I am just curious)

Reply:
Yeah. This is just there to avoid a magic number. For now, the design requires that you predetermine how many different query types there are. In the future, if of interest, you could mess with the code and get it to be more dynamic

Reply:
please add a word of comment